CONTRIBUTING.md

```diff
@@ -61,6 +61,7 @@ below:
 * Eleanor Smith (Met Office, UK)
 * Tomasz Trzeciak (Met Office, UK)
 * Mark Worsfold (Met Office, UK)
+* Shubhendra Singh Chauhan (DeepSource, India)
```

Comment:
Very happy for you to put your name with all the others in alphabetical ordering